c_cxx/README.md

```diff
@@ -7,56 +7,70 @@ This directory contains a few C/C++ sample applications for demoing onnxruntime
 5. OpenVINO_EP: Using OpenVino execution provider on the squeezenet model (TODO: Add CI build for it)
 6. opschema_lib_use (TODO: Add CI build for it)
 
-# How to build
+## How to build
+
+### Prerequisites
 
-## Prerequisites
 1. Visual Studio 2015/2017/2019
```

Comment:
I made more substantial changes to this file in my PR. Do you want to compare and see what should be taken from each?

Reply:
Sure. Why don't you merge your changes then I can resolve them with my branch?